pkg/cmd/server/plugin/plugin.go

```diff
@@ -45,6 +45,9 @@ func NewCommand(f client.Factory) *cobra.Command {
 		Hidden: true,
 		Short:  "INTERNAL COMMAND ONLY - not intended to be run directly by users",
 		Run: func(c *cobra.Command, args []string) {
+			config := pluginServer.GetConfig()
+			f.SetClientQPS(config.ClientQPS)
+			f.SetClientBurst(config.ClientBurst)
```

Comment:
How will this config be made available to additional BIA/RIAs? Or is the concern only this repo's built-in plugins?

Reply:
Looks like other plugins would have to call
```
config := pluginServer.GetConfig()
```
So you have to break up NewServer into a separate variable.

Cannot call functions after NewServer directly like below.
```
veleroplugin.NewServer().
		RegisterBackupItemAction("openshift.io/01-common-backup-plugin", newCommonBackupPlugin).
```

Seems fine enough :)

Reply:
For the out-tree plugins:
```go
package main

import (
	"github.com/vmware-tanzu/velero/pkg/cmd/server/config"
	veleroplugin "github.com/vmware-tanzu/velero/pkg/plugin/framework"
)

var config *config.Config

func main() {
	server := veleroplugin.NewServer()
	
        config = server.GetConfig()

	server.
		BindFlags(pflag.CommandLine).
		RegisterObjectStore("velero.io/azure", newAzureObjectStore).
		RegisterVolumeSnapshotter("velero.io/azure", newAzureVolumeSnapshotter).
		Serve()
}

// Then the code can read the velero server command args from the "config" variable

```